docs/incentivized-testnet-event/5-register-wallets.md

```diff
@@ -3,14 +3,21 @@ id: 5-register-wallet
 title: 5. Register Wallet
 ---
 
-## Register Wallet to Subnet
+In the `Makefile`, ensure `NETUID` and `SUBTENSOR_ENVIRONMENT` are set to the correct values.
 
-Register your `miner` to the default subnet (`1`):
+```
+NETUID = 1
```

Comment:
I'm confused here - isn't the netuid 2? @hide-on-bush-x can you have a look?

Reply:
Looks like it should be #1 according to https://github.com/masa-finance/issues/issues/194#issuecomment-2217320630